ops/testing.py

```diff
@@ -3658,10 +3689,6 @@ def _notify(
 
         Return a tuple of (notice_id, new_or_repeated).
         """
-        if type != pebble.NoticeType.CUSTOM:
```

Comment:
I'm curious about the decision between widening the allow-list vs. not having a list at all.

Reply:
Having the list there is annoying when adding new functionality - for example, the Juju PR goes first but has an ops charm in the tests, so has to patch all sorts of things. Basically, ops shouldn't be holding you back if Pebble has never functionality.

I don't really think we gain much by having a check here. If we want to warn people then it seems like static checking would be more approximate for this case.

@benhoyt might have more to say here: he added the check originally, but then also removed it in the PR (later reverted) that added support for change-updated-notice events.

Reply:
Nothing to add -- what Tony said makes sense (and is why I removed the check in the now-reverted PR).